notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -33,12 +33,12 @@
         "\n",
```

Comment:
<p>Add heading</p><p> </p><p>You can also -&gt; You can</p><p> </p><p>*We don't use the word also in our notebooks*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>done</p>